### DISCUSSION TITLE
How to make webpack to ignore a link tag in the html template?

### DISCUSSION THREAD

**Discussion by jackdawHome** [webpack/webpack#15181] [Q&A]
I have a link tag in my html template < link href="styles/bootstrap.min.css" rel="stylesheet">. I'm going to modify it dynamically to point at styles/bootstrap.rtl.min.css depending on user preferences. Both css are copied using CopyWebpackPlugin. Is there any way to prevent webpack5 from processing this link tag?

**Comment by alexander-akait** [comment]
Do you use `html-webpack-plugin`?

**Reply by jackdawHome** [reply]
Yes, i use html-webpack-plugin:
_new HtmlWebpackPlugin({
        filename: "read.html",
        template: "./src/taskpane/read.html",
        hash: true,
        chunks: ["scripts/polyfill", "helpers/bootstrap-icons", "scripts/CustomApp", "scripts/read", "bootstrap.min"],
      }),_
I would like to include a line: < link href="styles/bootstrap.min.css" rel="stylesheet"> in read.html, but I don't want webpack processing it. Is it possible?

**Reply by alexander-akait** [reply]
Please ask this in `html-webpack-plugin` repo, webpack doesn't have code for this

**Reply by jackdawHome** [reply]
Thank you, I will ask them
